### PR TITLE
Dev academy edits

### PR DESCRIPTION
**Pull request by elsehow**
This PR fixes minor copy-editing issues throughout the dev academy docs for stylistic consitancy and clarity. It also adds a bit more information on blockchains and the Cosmos ecosystem for novices who may be unfamiliar with core concepts (Tendermint, consensus, modules, the Cosmos SDK, etc).

### CODE REVIEW

File: dev-academy/basics/01-what-is-blockchain.md [hunk 1]
```diff
@@ -4,19 +4,15 @@ sidebar_position: 1
...
-- As new data comes in it is entered into a fresh block. Once the block is filled with data it is chained onto the
-  previous block, which makes the data chained together in chronological order.
-- Different types of information can be stored on a blockchain but the most common use so far has been as a ledger
-  for transactions.
- In Bitcoin’s case, blockchain is used in a decentralized way so that no single person or group has control—rather,
-  all users collectively retain control.
-- Decentralized blockchains are immutable, which means that the data entered is irreversible. For Bitcoin, this means
-  that transactions are permanently recorded and viewable to anyone.
+Specifically, new data comes in piece by piece in the form of *transactions*.
+Each transaction is added to a block. Once a block is full, the block is chained
```
Comment (orkunkl) [inline dev-academy/basics/01-what-is-blockchain.md]: `Once the block is full` This is not a requirement.

Comment (elsehow) [reply dev-academy/basics/01-what-is-blockchain.md]: resolved in the next commit

File: dev-academy/basics/01-what-is-blockchain.md [hunk 2]
```diff
@@ -4,19 +4,15 @@ sidebar_position: 1
...
-  previous block, which makes the data chained together in chronological order.
-- Different types of information can be stored on a blockchain but the most common use so far has been as a ledger
-  for transactions.
- In Bitcoin’s case, blockchain is used in a decentralized way so that no single person or group has control—rather,
-  all users collectively retain control.
-- Decentralized blockchains are immutable, which means that the data entered is irreversible. For Bitcoin, this means
-  that transactions are permanently recorded and viewable to anyone.
+Specifically, new data comes in piece by piece in the form of *transactions*.
+Each transaction is added to a block. Each blocks is chained to the previous
+block, creating a chronological ordering of data.
```
Comment (orkunkl) [inline dev-academy/basics/01-what-is-blockchain.md:11]: Let's give links to the merkle tree's in the next PR

**Review by orkunkl — APPROVED** [review]
Very good job! thanks.

**Pull request merged**